AMP-109903 Update docs for data permission restrictions

cyhk opened the pull request:
This adds a section in _Manage your Amplitude Data settings_ about the new Permissions Tab and its functionality.

markzegarelli commented on content/collections/data/en/amplitude-data-settings.md:
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
\ No newline at end of file
+Click _Save_ to implement any changes you make to your schema settings.
+
+## The Permissions Tab
```
```suggestion
## The Permissions tab
```

markzegarelli commented on content/collections/data/en/amplitude-data-settings.md:
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
+
+## The Permissions Tab
+
+Your company may want to limit the number of people who have access to perform Data Management functions. For example, you may want most of your users to have the Member role so they can create dashboards and charts but prefer that custom event creation and other data management features be limited to managers or administrators.
```
```suggestion
You may want to limit the number of people who have access to perform Data Management functions. For example, if you want most of your users to have the Member role so they can create dashboards and charts but limit custom event creation and other data management features to managers or administrators.
```

markzegarelli commented on content/collections/data/en/amplitude-data-settings.md:
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
+
+In the permissions tab, you can add additional restrictions for selected roles, limiting data management access while keeping other permissions for those roles.
+
+Under Restrict access in Amplitude Data, you can specify from the following options to restrict permissions to the viewer level on a per-project basis:
```
```suggestion
Under Restrict access in Amplitude Data, specify from the following options to restrict permissions to the viewer level on a per-project basis:
```
@cyhk Is this in Data settings? Will it be a new tab?

cyhk replied:
Yes, this will be a new tab in Data settings.

cyhk replied:
To clarify - the Permissions tab is a new tab in Data settings, Restrict access in Amplitude Data is the title of the bottom section on the page, which looks like this:
<img width="1034" alt="Screenshot 2024-10-18 at 16 50 34" src="https://github.com/user-attachments/assets/3fa11bc3-9cea-439c-9f40-79201688e1d0">

markzegarelli commented on content/collections/data/en/amplitude-data-settings.md:
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
+
+Then, click _Save_.
+
+The Permissions tab is only visible to admins and permission restrictions cannot be applied to admins themselves.
```
```suggestion
The Permissions tab is only visible to users with the Administrator role, and permission restrictions don't apply to Administrators.
```

markzegarelli commented on content/collections/data/en/amplitude-data-settings.md:
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
+
+The Permissions tab is only visible to admins and permission restrictions cannot be applied to admins themselves.
+
+### Copy to Other Projects
```
```suggestion
### Copy to other projects
```

markzegarelli commented on content/collections/data/en/amplitude-data-settings.md:
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
+
+### Copy to Other Projects
+
+You can also copy current permission restriction settings to other projects by clicking on the _Copy to Other Projects_ button. To copy settings, select the desired projects and click _Save_.
```
```suggestion
Click _Copy to Other Projects_ to apply the current permission restriction settings to another project.
```

cyhk commented:
@markzegarelli - asking for a re-review so I don't slip anything past unchecked, but I made some changes based on your suggestions:
- slightly more extensive tweaking for the first two paragraphs
- removed the part about permission restrictions not applying to admins and added a part about the feature being available for enterprise customers only for the last paragraph before the *Copy to other projects* section.

markzegarelli commented:
Thanks @cyhk! I had automerge enabled on this, so it merged after I approved and resolved conversations. Is that OK?
